samples/javacv-demo-kotlin/build.gradle.kts

```diff
@@ -0,0 +1,35 @@
+plugins {
+    // Apply the Kotlin JVM plugin to add support for Kotlin.
+    kotlin("jvm").version("1.3.72")
+    // Apply the java-library plugin for API and implementation separation.
+    `java-library`
+    id("org.bytedeco.gradle-javacpp-platform").version("1.5.4-SNAPSHOT")
+}
+
+repositories {
+    jcenter()
+    maven { url = uri("https://oss.sonatype.org/content/repositories/snapshots/") }
```

Comment:
This isn't going to work for users building JavaCV locally or using a release version.

Reply:
Good point. Added `mavenLocal()`